src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
+
+import style from './ProgressBar.module.css';
+
+type Props = {
+  percentage: number;
+  textPosition: 'in' | 'up';
+  textAdded?: string;
+};
+
+function getColor(percentage: number) {
+  const red = '#D76B65';
+  const blue = '#2D689F';
+  const green = '#37A169';
+
```

Comment:
After seeing the progress bar on action, I think we should only have 2 colors, blue for less than 100% and green for 100%. Having red might be perceived as if the student has done something wrong when it reality he just hasn't completed the topic yet.